### PR TITLE
Fix/256

### LINKED ISSUE #256: Fix continuous integration
Whether by fixing Travis or switching to Github Actions.

### PR DESCRIPTION
**Pull request by omangin**
Migrate from Travis to Github actions.

- [x] Create github action
- [x] Include install test and condition to being on schedule
- [x] Remove travis file
- [ ] Check that install test runs on schedule (can be done only after merge)
- [x] Add badge to wiki
- [x] Remove remaining travis config (webhook etc.)

Fixes #256.

### CODE REVIEW

File: tests/test_git.py [hunk 1]
```diff
@@ -72,6 +83,7 @@ def test_git(self):
         # self.assertEqual(hash_i, hash_j)
 
     def test_manual(self):
+        print(self.default_pubs_dir)
```
Comment (benureau) [inline tests/test_git.py:86]: Should that be there?

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - cron: '0 8 * * *'
+
+
+jobs:
+  unit-test:
+    name: Run unit tests
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```
Comment (benureau) [inline .github/workflows/tests.yaml:16]: We are not testing Python 2.7 anymore. Should be start updating the readme to remove mention of Python 2 support, and then actually remove it in a future release?

**Review by benureau — APPROVED** [review]

**Comment by benureau** [comment]
Nicely done!

File: tests/test_git.py (hunk 1 above)
Comment (omangin) [reply tests/test_git.py:86]: Good catch! Thanks.

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (omangin) [reply .github/workflows/tests.yaml:16]: Actually the readme already states that it is not supported any more as we had already stopped testing for it some time ago. We could indeed remove all mentions of it with the next release. I will leave it for now anyway.

**Pull request merged**